locale/en/blog/release/v4.1.1.md

```diff
@@ -2,7 +2,7 @@
 date: 2015-09-23T02:01:38.545Z
 version: 4.1.1
 category: release
-title: Node v4.1.1 (Stable)
+title: Node v4.1.1 (Current)
```

Comment:
That's why I did this in a separate commit, so the others can be cherry-picked, if needed.